gallery/src/pages/components/ha-selector.ts

```diff
@@ -166,7 +166,19 @@ const SCHEMAS: {
       object: { name: "Object", selector: { object: {} } },
       select_radio: {
         name: "Select (Radio)",
-        selector: { select: { options: ["Option 1", "Option 2"] } },
+        selector: {
+          select: { options: ["Option 1", "Option 2"], mode: "list" },
+        },
+      },
+      select_checkbox: {
+        name: "Select (Checkbox)",
+        required: false,
```

Comment:
Is `required` what changes it between radio and checkbox? Because there is another thing and that is that checkboxes allow you to select multiple. That's not reflected in the selector options here?